### ISSUE TITLE
Delete sample employee data

### ISSUE THREAD

**Issue by mahavidhan** [sapplica/sentrifugo#68]
Hello.
What I have learnt so far from my research is that there is no way to remove an employee from the system and I fell thats the right way to do it. However, can we delete few sample data entered by the name test, sample etc? Either from within the interface or via backend? I ended up adding lots of such sample test records which is not required when the system goes live in our office and I want to get rid of that. Please advice.

**Comment by benhyman86** [comment]
I'm in exactly the same boat. Been testing the system prior to implementation, ready to go - just need to remove all the sample employees i've been testing with.
There really should be a way super-admin can remove employee data.

**Comment by eljaunis** [comment]
You can truncate the database table that contains EMP data, maybe?

**Comment by benhyman86** [comment]
yes, i’ve found what i need in the database tables, but there’s obviously the concern that suddenly there’s relational bits that suddenly become disjoint…?

**Comment by eljaunis** [comment]
Can't you check Db relationships using phpMyAdmin?

**Comment by 4poorv** [comment]
I tried to delete records using phpmyadmin. looks like there are multiple tables from which records have to be deleted as '**main_users**', '**main_employees**' and '**main_employees_summary**' etc.
I think there should be provision (At least for Super Admin) to delete employee records.

**Comment by skart76** [comment]
Are there any updates related to this topic or is it still the case that the employee data cannot be deleted?
Do you have a sql scrip that would remove the data from the db?
